### PR TITLE
Move letters into separate YML file

### PR DESCRIPTION
**Pull request by colinbruce**
This will simplify the main YML document and make it easier 
for the content team to manage the letters in isolation.

### CODE REVIEW

**Comment by dotemacs** [comment]
Looks fine to me

File: app/views/evidence/_confirmation_part_remission.html.slim [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-         date: confirmation.expires_at.strftime(Date::DATE_FORMATS[:gov_uk_long]))
+         reference: confirmation.application.reference,
+         user_name: current_user.name,
+         part_payment_amount: "£#{confirmation.amount_to_pay}",
+         total_income: "£#{confirmation.income}",
+         children: confirmation.application.children,
+         date_for_part_payment: confirmation.expires_at.strftime(Date::DATE_FORMATS[:gov_uk_long]),
+         name_delivery_manager: 'YOUR MANAGERS NAME',
+         court_tribunal_name: 'APPEAL COURT NAME',
+         date_for_appeal: 'INSERT DATE TO APPEAL')
```
Comment (colinbruce) [inline app/views/evidence/_confirmation_part_remission.html.slim]: @leighmoney I set the text for these three fields to address my comment below...

File: config/locales/letters_en.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        <p><strong>How to pay</strong></p> 
+        <ul> 
+          <li>by post with a cheque written out to ‘HM Courts and Tribunals Service’ for the amount</li>
+          <li>by debit card over the phone</li> 
+          <li>in person with a cheque, cash or debit card at the court counter</li>
+        <br> 
+        <p>Please make sure you pay this amount by %{date_for_part_payment}</p>
+        <br>
+        <p><strong>If you don’t agree with the amount you have to pay</strong></p>  
+        <p>If you disagree with our decision about the amount you have to pay, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this by %{date_for_appeal}.</p>
```
Comment (colinbruce) [inline config/locales/letters_en.yml]: @leighmoney this was the only section I couldn't further match.  %{name_delivery_manager}, %{court_tribunal_name} and %{date_for_appeal} are not currently stored/generated, so I set the text above...

**Comment by colinbruce** [comment]
@leighmoney I got everything merged and the tests to pass, have a look at my comments above and we can chat on Thursday if you want further changes.

**Pull request merged**